VERSION

```diff
@@ -1 +1 @@
-0.0.20
+0.0.21
```

Comment:
I'll cut a first release soon (probably version 0.1.0, or maybe just 1.0.0) once I have Travis building and deploying prebuilt binaries - as such no need to bump this.

Reply:
Fixed - feel free to squash on merge.